ipam/store.go

```diff
@@ -72,8 +72,8 @@ func (ipamStore *ipamStore) addVm(stride uint, vm *Vm) error {
 	vm.EffectiveSeq = effectiveVmSeq
 	ipamVm := IpamVm{Vm: *vm}
 	tx.NewRecord(ipamVm)
-	tx.Create(&ipamVm)
-	err := common.MakeMultiError(ipamStore.DbStore.Db.GetErrors())
+	tx = tx.Create(&ipamVm)
```

Comment:
This line's cool in theory but I'm going to make a test. @cgilmour and I already connected on this.